lib/galaxy/util/xml_util.py

```diff
@@ -137,8 +137,9 @@ def parse_xml( file_name, preserve_comments=False ):
             else:
                 tree = XmlET.parse( fh )
         except Exception, e:
-            log.exception( str( e ) )
-            return None, e
+            error_message = "Exception attempting to parse %s: %s" % ( str( file_name ), str( e ) )
```

Comment:
You mean for `e`? Probably not, but it was just a revert, so I left it as it was.